### PR TITLE
add most recent amip to pipeline

### LINKED ISSUE #881: Set up a pipeline for the best AMIP we have

### PR DESCRIPTION
**Pull request by szy21**
## Purpose 

Adds a job for the best AMIP we have to the pipeline. We can run it every other week?

Closes #881 

## Content

----
- [ ] I have read and checked the items on the review checklist.

### CODE REVIEW

File: config/amip_configs/amip.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+FLOAT_TYPE: "Float32"
+albedo_model: "CouplerAlbedo"
+anim: false
+atmos_config_file: "config/longrun_configs/amip_target_diagedmf.yml"
```
Comment (szy21) [inline config/amip_configs/amip.yml:4]: Would it be better to just have all the configs here, instead of inheriting from an atmos job?

**Comment by Sbozzolo** [comment]
I was going to add a new pipeline for this on Derecho, but it's definetely easier on clima.

I think we can start with running it weekly, and it gets in the way, we can move it biweekly.

**Comment by szy21** [comment]
> I was going to add a new pipeline for this on Derecho, but it's definetely easier on clima.
> 
> I think we can start with running it weekly, and it gets in the way, we can move it biweekly.

Sounds good. I don't quite remember how to set up the scheduled pipeline. Do we need to merge this PR first, and then set it up on buildkite?

File: .buildkite/amip/pipeline.yml [hunk 2]
```diff
@@ -0,0 +1,95 @@
+agents:
+  queue: clima
+  slurm_time: 72:00:00
+  modules: climacommon/2024_05_27
+
+env:
+  JULIA_LOAD_PATH: "${JULIA_LOAD_PATH}:${BUILDKITE_BUILD_CHECKOUT_PATH}/.buildkite"
+  OPENBLAS_NUM_THREADS: 1
+  JULIA_NVTX_CALLBACKS: gc
+  OMPI_MCA_opal_warn_on_missing_libcuda: 0
```
Comment (Sbozzolo) [inline .buildkite/amip/pipeline.yml:10]: This is not needed

**Comment by Sbozzolo** [comment]
> > I was going to add a new pipeline for this on Derecho, but it's definetely easier on clima.
> > I think we can start with running it weekly, and it gets in the way, we can move it biweekly.
> 
> Sounds good. I don't quite remember how to set up the scheduled pipeline. Do we need to merge this PR first, and then set it up on buildkite?

I can create it for you

**Comment by Sbozzolo** [comment]
> > > I was going to add a new pipeline for this on Derecho, but it's definetely easier on clima.
> > > I think we can start with running it weekly, and it gets in the way, we can move it biweekly.
> > 
> > 
> > Sounds good. I don't quite remember how to set up the scheduled pipeline. Do we need to merge this PR first, and then set it up on buildkite?
> 
> I can create it for you

Scheduled to run on Tuesdays at 4.05 AM

**Review by Sbozzolo — APPROVED** [review]

**Comment by szy21** [comment]
> > > > I was going to add a new pipeline for this on Derecho, but it's definetely easier on clima.
> > > > I think we can start with running it weekly, and it gets in the way, we can move it biweekly.
> > > 
> > > 
> > > Sounds good. I don't quite remember how to set up the scheduled pipeline. Do we need to merge this PR first, and then set it up on buildkite?
> > 
> > 
> > I can create it for you
> 
> Scheduled to run on Tuesdays at 4.05 AM, feel free to change it. It's climacoupler-amip.

Thanks!

**Pull request merged**